### PR TITLE
Added inputs

### PR DESCRIPTION
**Pull request by MikaelWeidenhielm**
ALBEEE kolla mina inputs, dom blev jättefina tycker jag. Haaej!

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -45,6 +47,65 @@
...
+      </div>
+
+    </div>
+
+    <br><br>
+
+      <div class="dimensions__field">
+
+        <h2>Fields</h2>
+
```
Comment (eskilfogelstrom) [inline src/index.html]: This is a nice example of when something consists of multiple components. First we have a list of things (could be anything). Within that list we have some draggable items.
```html
<div class="list">
  <div class="list__item draggable">...</div>
</div>
```
Each item also has two modifiers. When it's being dragged and a placeholder for the item being dragged. To preview how it looks, make a list like this:
```html
<div class="list">
  <div class="list__item draggable">...</div><!-- Normal -->
  <div class="list__item draggable--dragged">...</div><!-- When it's being dragged -->
  <div class="list__item draggable--placeholder">...</div><!-- Placeholder for the element being dragged -->
</div>
```

File: src/index.html [hunk 2]
```diff
@@ -45,6 +47,65 @@
...
+        <div class="dragable">Years</div>
+        <div class="dragable">Foo</div>
+        <div class="dragable">Foo</div>
+
+
+      </div>
+
+      <br>
+
+      <div class="dimensions__variables">
```
Comment (eskilfogelstrom) [inline src/index.html]: Same thing here with multiple components. The boxes where you drop stuff can be considered cards and should then use the `.card` class. The area where you actually drop things is a separate class and is essentially the same as the `.draggable--placeholder` above.

File: src/styles/atoms/_buttons.scss [hunk 3]
```diff
@@ -3,8 +3,15 @@
   border: 0;
   border-radius: $border-radius;
   font-family: 'Brandon Grotesque', sans-serif;
-  font-size: 15px;
   color: white;
+  font-size: $font-size;
+  outline: none;
+}
+
+.button__input{
```
Comment (eskilfogelstrom) [inline src/styles/atoms/_buttons.scss]: This can be removed no? :trollface:

File: src/styles/atoms/_draggable.scss [hunk 4]
```diff
@@ -0,0 +1,23 @@
+.draggable{
+  background-color: $color-tertiary-3;
+  border-radius: 5px;
+  box-shadow: 2px 2px 5px rgba(#000, 0.15);
+  color: #fff;
+  font-size: $font-size-l;
```
Comment (eskilfogelstrom) [inline src/styles/atoms/_draggable.scss]: Not sure if the font size should be so large. What do you think?

File: src/styles/atoms/_draggable.scss [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  text-align: center;
+  cursor: move;
```
Comment (eskilfogelstrom) [inline src/styles/atoms/_draggable.scss]: This gets overwritten below so you can just remove it

File: src/styles/atoms/_draggable.scss [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  cursor: -webkit-grab;
```
Comment (eskilfogelstrom) [inline src/styles/atoms/_draggable.scss]: Can only be `grab`

File: src/styles/atoms/_draggable.scss [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+  &:active{
+    cursor: -webkit-grabbing;
```
Comment (eskilfogelstrom) [inline src/styles/atoms/_draggable.scss]: ☝️  `grabbing`

File: src/styles/atoms/_inputs.scss [hunk 8]
```diff
@@ -0,0 +1,36 @@
+.title__input{
```
Comment (eskilfogelstrom) [inline src/styles/atoms/_inputs.scss]: Remove this file

File: src/styles/atoms/_list__item.scss [hunk 9]
```diff
@@ -0,0 +1,25 @@
+.list__item{
```
Comment (eskilfogelstrom) [inline src/styles/atoms/_list__item.scss]: Remove this file

File: src/styles/atoms/_typography.scss [hunk 10]
```diff
@@ -20,4 +20,9 @@ h3 {
 }
 p {
   font-weight: 400;
+  color: grey;
+}
+
+.list__title{
```
Comment (eskilfogelstrom) [inline src/styles/atoms/_typography.scss]: Remove since it's not used

**Review by eskilfogelstrom — CHANGES_REQUESTED** [review]

File: src/styles/atoms/_draggable.scss (hunk 5 above)
Comment (MikaelWeidenhielm) [reply src/styles/atoms/_draggable.scss]: Shady shit happened when I tried to remove this, I'll just wait with it

File: src/styles/atoms/_draggable.scss (hunk 6 above)
Comment (MikaelWeidenhielm) [reply src/styles/atoms/_draggable.scss]: Doesn't actually respond to `grab` for some reason

**Review by eskilfogelstrom — APPROVED** [review]

**Comment by eskilfogelstrom** [comment]
Nice job micke2000

**Pull request merged**